### PR TITLE
OpenWhisk chokes on SVG if not base64 encoded

### LINKED ISSUE #507: OpenWhisk chokes on SVG if not base64 encoded
**Description**
This was found in https://github.com/adobe/developer.adobe.com/issues/88: Runtime/OpenWhisk will throw a 400 on SVG files spooled by the Helix static action. [Apparently](https://doc.akka.io/api/akka-http/10.0.4/akka/http/scaladsl/model/MediaTypes$.html#image/svg+xml:akka.http.scaladsl.model.MediaType.Binary), OpenWhisk will treat everything not explicitly containing `text` in its content-type as binary. Even though SVG is plain text in nature, it is expected to come base64 encoded.

**To Reproduce**
1. `hlx demo --type=full`
2. `hlx deploy`
3. Go to https://adobeioruntime.net/api/v1/web/<your_wsk_namespace>/default/hlx--static?owner=adobe&repo=developer.adobe.com&ref=master&strain=default&entry=/icons/spectrum-css-icons.svg&root=/htdocs&plain=true
4. See 400 error:
```
{
  "code": "<id>",
  "error": "Response type in header did not match generated content type."
}
```

**Expected behavior**
SVG files should be delivered with 200 and intact

**Version:**
0.13.3-pre.2

### PR DESCRIPTION
**Pull request by rofe**
Fixes #507 
Fixes https://github.com/adobe/developer.adobe.com/issues/88

### CODE REVIEW

File: src/openwhisk/static.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function isBinary(type) {
     return false;
   }
   if (type.match(/.*\/.*xml/)) {
-    return false;
+    return !!type.match(/svg/); // openwshisk treats svg as binary
```
Comment (tripodsan) [inline src/openwhisk/static.js]: I think openwhisk treats all `image/*` as binary. maybe that would be the easier fix.

Comment (rofe) [reply src/openwhisk/static.js]: So if I understand you correctly, you mean adding a
```
if (type.match(/image\/.*/)) {
    return true;
}
```
above?

Comment (rofe) [reply src/openwhisk/static.js]: or rather wrap all the existing `if`s in a 
```
if (!type.match(/image\/.*/)) {
    ...
}
```

Comment (tripodsan) [reply src/openwhisk/static.js]: we should use the same code as openwhisk. I think it's here:
https://github.com/apache/incubator-openwhisk/blob/master/core/controller/src/main/scala/org/apache/openwhisk/core/controller/WebActions.scala#L325

but never mind for now.

**Pull request merged**